Clarify requests on Private DDN

tomhasura opened the pull request:
## Description 📝

Clarify default requests that are set on Private DDN

## Quick Links 🚀

 <!-- Links to the relevant place(s) in the CloudFlare Pages build. Wait for CF comment for link. -->

## Assertion Tests 🤖

<!-- Add assertions between the comments below to have ChatGPT check the quality of your docs contribution (Diff) and 
how well it integrates with existing docs. E.g., A user should be able to easily understand how to make a simple 
query. -->

<!-- For more info, see the Action's docs in the marketplace: https://github.com/marketplace/actions/docs-assertion-tester#usage -->

<!-- DX:Assertion-start -->
<!-- DX:Assertion-end -->

arvi3411301 commented:
I've opened up another [PR](https://github.com/hasura/ddn-docs/pull/959) for the same

tomhasura commented:
> I've opened up another [PR](https://github.com/hasura/ddn-docs/pull/959) for the same. I've removed the public ddn sub headings since the page is under the private ddn. Added a note about the same

Thanks.  I'll close my PR in favor of yours
